docs-ref-conceptual/Latest-version/includes/cli-install-linux-script.md

```diff
@@ -66,48 +88,23 @@ installation. The default install location is the user's home directory ($HOME).
 Here are some common problems seen during a manual installation. If you experience a problem not
 covered here, [file an issue on GitHub][03].
 
-### Install without Python 3
+- **`az: command not found`**
 
-The Azure CLI has dropped support for Python 2.7 since version [2.1.0][02]. On your system, there
-may be a Python version that predates the requirement of Python 3.6.x. Find a replacement `python3`
-package.
+  - Make sure `~/bin` is on your `PATH` (`echo $PATH`), the symlink exists (`ls -l ~/bin/az`), then
+    `hash -r` and reopen your shell.
 
-### curl "Object Moved" error
-
-If you get an error from `curl` related to the `-L` parameter, or an error message including the
-text "Object Moved", try using the full URL instead of the `aka.ms` redirect:
-
-```bash
-curl https://azurecliprod.blob.core.windows.net/install | bash
-```
-
-### `az` command not found
-
-If you can't run the command after installation using `bash` or `zsh`, try clearing your shell's
-command hash cache and check if the problem is resolved.
-
-```bash
-hash -r
-```
-
-The issue can also occur if you didn't restart your shell after installation. Make sure that the
-location of the `az` command is in your `$PATH`. The location of the `az` command is ...
-
-```
- <install path>/bin
-```
+- **SSL/crypto errors**
 
-### Proxy blocks connection
+  - Ensure OpenSSL and libffi are installed for your distro and that your Python build links against
+    them.
 
-[!INCLUDE[configure-proxy](configure-proxy.md)]
+- **Proxy issues**
 
-In order to get the installation scripts, your proxy needs to allow HTTPS connections to the
-following addresses:
+  - Configure pip or environment variables (for example, `HTTPS_PROXY`) and allow access to:
 
-- `https://aka.ms/`
-- `https://azurecliprod.blob.core.windows.net/`
-- `https://pypi.python.org`
-- Endpoints used by your distribution's package manager (if any) for core packages
+    - `https://pypi.org/`
+    - `https://files.pythonhosted.org/`
+    - Any endpoints used by your distro's package manager (for Python/venv prerequisites)
```

Comment:
Why not link to https://learn.microsoft.com/en-us/cli/azure/azure-cli-endpoints?view=azure-cli-latest&tabs=azure-cloud